src/main/java/com/plaid/client/response/IdentityGetResponse.java

```diff
@@ -46,15 +46,33 @@ public ItemStatus getItem() {
   }
 
   public static final class Address {
-    private Boolean primary;
     private AddressData data;
+    private boolean primary;
 
-    public Boolean isPrimary() {
+    public AddressData getData() {
+      return data;
+    }
+
+    public boolean isPrimary() {
       return primary;
     }
 
-    public AddressData getData() {
-      return data;
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
out of curiousity, how did we know to add this?

Reply:
`Address` class in `AssetReportGetResponse` has this method https://github.com/plaid/plaid-java/pull/164/files#diff-0ba440f50f0fdf563a7271418652d2aeL384